### ISSUE TITLE
[a11y] 9.1.3.2 - Logical order

### ISSUE THREAD

**Issue by tbsbdr** [owncloud/web#5383]
- Context: The Web UI should be accessible according to the WCAG2.1 /BITV 2.0 guidelines. The number in the title relates to the test-steps: https://www.bitvtest.de/bitv_test/das_testverfahren_im_detail/pruefschritte.html
- "Status" relates to the evaluation scheme https://www.bitvtest.de/bitv_test/das_testverfahren_im_detail/verfahren.html#c1262 
- "a11y" is short for accessibility
- ----------------

## General notes
- **Pages 1 to 3:** Above the h1-heading is a visually hidden text which tells the users if the elements in the table are selected or not, e.g. "Keine Elemente ausgewählt.". This text should be placed after the h1-heading, otherwise it could be missed out by users when navigating via headings.

## Status: rather fulfilled

**Comment by kulmann** [comment]
Yes, already solved, that's why it's listed in https://github.com/owncloud/web/pull/5512 ;-)

**Comment by kulmann** [comment]
We'll only actually close the tickets when the PRs get merged.

**Comment by kulmann** [comment]
Maybe we didn't mention it in the confluence page though? not sure 🤔

**Comment by stevefrenzel** [comment]
> Maybe we didn't mention it in the confluence page though? not sure 🤔

My bad, I skipped a row on the confluence page. Updated the colors so it's easier to see what still needs to be done.
